01-node-tutorial/answers/10-fs-sync.js

```diff
@@ -0,0 +1,24 @@
+const { readFileSync, writeFileSync } = require('fs')
+
+const firstFile = readFileSync('./temporary/.keep', 'utf8')
+
+const firstLine = 'Good Afternoon'
+const secondLine = 'Time to go to sleep'
+const thirdLine = 'Goodnight'
+
+writeFileSync(
```

Comment:
Call writeFileSync 3 times (once for each of the lines).  Also make sure you read the file after that to confirm it worked!